i18n/api/en.ts

```diff
@@ -338,19 +338,19 @@ const en: Translations = {
     },
     oasDeferralAvailable: {
       heading: 'You may be able to defer your payments',
-      text: 'To learn more about your option to delay your first Old Age Security pension payment, {LINK_OAS_DEFER_CLICK_HERE}.',
+      text: 'Learn more about the option to delay your first Old Age Security pension payment, {LINK_OAS_DEFER_CLICK_HERE}.',
```

Comment:
the underline style should start from "option", also there is no "click here" in the description